src/tracks/SampleTrack.cpp

```diff
@@ -335,6 +346,9 @@ SampleTCOView::SampleTCOView( SampleTCO * _tco, TrackView * _tv ) :
 	connect( m_tco, SIGNAL( sampleChanged() ),
 			this, SLOT( updateSample() ) );
 
+	connect(m_tco, SIGNAL(wasReversed()),
+			this, SLOT(update()));
```

Comment:
This doesn't line up unless tabs are 4 spaces, might as well just make it a single line.
```suggestion
	connect(m_tco, SIGNAL(wasReversed()), this, SLOT(update()));
```